Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -485,7 +485,7 @@ public async Task ProcessExecutionResult(SubmissionExecutionResult submissionExe
             this.submissionsData.Update(submission);
         }
 
-        await this.submissionsForProcessingData.MarkProcessed(submission.Id);
+        await this.submissionsForProcessingData.MarkProcessedAndSerializeWorkerResult(submissionExecutionResult.Map<SubmissionExecutionResultServiceModel>());
```

Comment:
Better keep naming of the method to be consistent with other methods (MarkProcessing etc). Pass serialized execution result.

Reply:
Name reverted.